app_dart/cloudbuild_app_dart.yaml

```diff
@@ -6,7 +6,7 @@
 
 steps:
   # Build dashboard.
-  - name: gcr.io/$PROJECT_ID/flutter
+  - name: us-docker.pkg.dev/$PROJECT_ID/gcr.io/$PROJECT_ID/flutter
```

Comment:
Shall this be: `us-docker.pkg.dev/$PROJECT_ID/gcr.io/flutter`

Reply:
Yes, that would be simpler. We will need to also change the `flutter_image_build.yaml` to upload with that name, but that change will happen after this one regardless.

Reply:
We copied over the image with the simplified name and are running a test to pull it down with that path